src/internal/disableLocalStorage.ts

```diff
@@ -0,0 +1,30 @@
+export function disableLocalStorage() {
+  if (!globalThis.localStorage) return
```

Comment:
Changed in [`4a97ce8` (#46)](https://github.com/rune/rune-games-sdk/pull/46/commits/4a97ce81f80920252be0c0beed67a286d7ba4839)

Reply:
> Both storages use the same Storage object, so this is actually disabling both (because I am changing the prototype). I'll just rename the method to disableStorage.

Good point! Would be great to add comment in the code where you `Object.defineProperties` because it is not clear from that this will disable both localStorage and sessionStorage.

Reply:
Added a comment explaining it.